R/Report_KRI.R

```diff
@@ -56,21 +67,22 @@ Report_KRI <- function(
       GroupLevel <- gsub("[^[:alnum:]]", "", GroupLevel)
       SnapshotDate <- gsub("[^[:alnum:]]", "", as.character(SnapshotDate))
 
-      strOutpath <- paste0(getwd(), "/kri_report_", StudyID, "_", GroupLevel, "_", SnapshotDate, ".html")
+      strOutpath <- file.path(strOutputDir, paste0("kri_report_", StudyID, "_", GroupLevel, "_", SnapshotDate, ".html"))
     } else {
-      strOutpath <- paste0(getwd(), "/kri_report.html")
+      strOutpath <- file.path(strOutputDir, "kri_report.html")
     }
+  } else {
+    strOutpath <- file.path(strOutputDir, strOutputFile)
   }
 
-  rmarkdown::render(
-    system.file("report", "Report_KRI.Rmd", package = "gsm"),
+  render_rmd(
+    input = system.file("report", "Report_KRI.Rmd", package = "gsm"),
     output_file = strOutpath,
     params = list(
```

Comment:
```suggestion
    strInputPath = system.file("report", "Report_KRI.Rmd", package = "gsm"),
    strOutputFile = strOutputFile,
    strOutputDir = strOutputDir,
    lParams = list(
```
Since we're exporting this, we should use our naming convention (more on this below).